### ISSUE TITLE
An virtfs(9p) backend error occurs when running python when SMP=1.

### ISSUE THREAD

**Issue by ken4647** [syswonder/ruxos#132]
when running cmd `make A=apps/c/python3 ARCH=aarch64 V9P=y NET=y MUSL=y LOG=off run` :
```bash
qemu-system-aarch64: Failed to decode VirtFS request type 50
```
and type 50 means fsync.

The backend error is like this:
![8854aa9d7e2be0db8bc2af91ef0bc86](https://github.com/syswonder/ruxos/assets/87317372/21d1b959-3873-4026-9cc4-ccb0213f027f)

**Comment by ken4647** [comment]
in ![9P2000.L](https://github.com/chaos/diod/blob/master/protocol.md) , the format of `tfsync` is like below:

#### fsync - flush any cached data to disk
```
size[4] Tfsync tag[2] fid[4]
size[4] Rfsync tag[2]
```
fsync tells the server to flush any cached data associated with fid, previously opened with lopen.

However, 9p client in linux, it is implement like this:
```
size[4] Tfsync tag[2] fid[4] datasync[4]
```
This may be due to some problems or requirements that have caused the operation specifications of 9p's fsync to change, so our interface also needs to be patched accordingly.

**Comment by ken4647** [comment]
fixed in #133 .
